packages/command-palette/src/index.tsx

```diff
@@ -119,8 +121,9 @@ export function CommandMenuGroup( {
 		search,
 		navigate,
 		useCommands,
-		wpcom,
 		currentRoute,
+		useSites,
+		userCapabilities,
```

Comment:
Maybe we should think of a way to improve the existing architecture instead of just drilling more props all over.

Reply:
Suggestions are more than welcome. I don't like this either, but I really couldn't think of a better alternative. 

My fear is that thinking and finding a proper solution at this point will take a decent chunk of time, so I opted for something that is quirky but it works. We can make things right later.